tracker/src/plausible.js

```diff
@@ -341,7 +341,7 @@
       }
     }
 
-    if (document.visibilityState === 'prerender') {
+    if (document.visibilityState === 'hidden' || document.visibilityState === 'prerender') {
```

Comment:
I noticed a tiny improvement that would save a few bytes in the compiled script: 
```suggestion
    if (['hidden', 'prerender'].includes(document.visibilityState)) {
```

Reply:
https://github.com/plausible/analytics/pull/5344#discussion_r2048708553 Sorry - we should not use Array.includes in tracker.